model/change_point_detection.go

```diff
@@ -428,3 +434,42 @@ func createChangePoint(ctx context.Context, env cedar.Environment, resultToUpdat
 	_, err := env.GetDB().Collection(perfResultCollection).UpdateOne(ctx, filter, update)
 	return errors.Wrap(err, "Unable to create change point")
 }
+
+func TriageChangePoints(ctx context.Context, env cedar.Environment, changePoints map[string]string, status TriageStatus) error {
+	session, err := env.GetDB().Client().StartSession()
+	if err != nil {
+		return errors.Wrap(err, "Unable to start session for triage update")
+	}
+	defer session.EndSession(ctx)
+	if err := session.StartTransaction(); err != nil {
+		return errors.Wrap(err, "Unable to start transaction for triage update")
+	}
+	return mongo.WithSession(ctx, session, func(sc mongo.SessionContext) error {
+		for perfResultId, measurement := range changePoints {
```

Comment:
how many change points do we expect there to be typically and maximally?

Reply:
I think the build barons typically triage all of them in a given version using multiselect, and I'm guessing ~10 per version?

So typically probably ~10 at a time, unless they select multiple versions --> then it would be 10*verions, which for a given page size they are using would max out around ~10*page_size. Page size would probably be 50, so maximally ~500 in typical situations.